tools/text_processing_deployment/sh_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Why do we need a separate `sh_test.sh`? why export_grammars.sh with MODE!="test" is not enough?

Reply:
Jenkins tests already launch in a docker. We cannot launch another docker from within a docker in Jenkins.